src/main/java/Duke.java

```diff
@@ -1,10 +1,208 @@
+import java.util.Scanner;
+import java.util.*;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.File;
 public class Duke {
+//    private static final Task[] listTasks = new Task[100];
+    private static final ArrayList<Task> listTasks= new ArrayList<>(1);
+    public static int jobCount = 0;
+
+    /**
+     * Function to print Logo and Hello
+     */
+    static void printHello() {
+        String logo = "   _____  .__        \n"
+                + "  /  _  \\ |__|______ \n"
+                + " /  /_\\  \\|  \\_  __ \\\n"
+                + "/    |    \\  ||  | \\/\n"
+                + "\\____|__  /__||__|   \n"
+                + "        \\/           \n";
+        System.out.println("\tHello from\n" + logo + "\n");
+        System.out.println("____________________________________________________________\n");
+        System.out.println("\tSup! I'm Air\n" + "\tHow can I help you out today?\n");
+        System.out.println("____________________________________________________________\n");
+    }
+    private static void writeToFile(String filePath, String textToAdd) throws IOException {
+        FileWriter fw = new FileWriter(filePath);
```

Comment:
Consider using more descriptive variable names.
Longer names should not be avoid since the IDE would autocomplete them for you.
(That said, don't go too extreme and use **very** long and cumbersome variable names)